apps/src/netsim/NetSimRouterNode.js

```diff
@@ -710,12 +917,6 @@ NetSimRouterNode.prototype.onNodeTableChange_ = function (rows) {
   }
 };
 
-NetSimRouterNode.prototype.onMyStateChange_ = function (remoteRow) {
-  this.dnsMode = remoteRow.dnsMode;
-  this.dnsNodeID = remoteRow.dnsNodeID;
-  this.stateChange.notifyObservers(this);
-};
```

Comment:
Moved adjacent to `NetSimRouterNode.prototype.buildRow_` since this method should be a parallel to that out (serialize out/serialize in).